actionpack/lib/action_dispatch/system_testing/browser.rb

```diff
@@ -26,6 +26,8 @@ def options
           headless_chrome_browser_options
         when :headless_firefox
           headless_firefox_browser_options
+        when :chrome
+          chrome_browser_options
```

Comment:
What if we generalized the solution?

```suggestion
        else
          capabilities
```

Reply:
Yeah - that's a good idea. It'll be `nil` anyway for browsers other than Chrome / Firefox.